dashboard/tests/conftest.py

```diff
@@ -52,3 +52,14 @@ def small_event_line_limit():
     os.environ["EVENT_READ_LINE_LENGTH_LIMIT"] = "1024"
     yield 1024
     os.environ.pop("EVENT_READ_LINE_LENGTH_LIMIT", None)
+
+
+@pytest.fixture
+def fast_gcs_failure_detection():
+    os.environ["GCS_CHECK_ALIVE_MAX_COUNT_OF_RPC_ERROR"] = "2"
+    os.environ["GCS_CHECK_ALIVE_INTERVAL_SECONDS"] = "1"
+    os.environ["GCS_RETRY_CONNECT_INTERVAL_SECONDS"] = "1"
+    yield
+    os.environ.pop("GCS_CHECK_ALIVE_MAX_COUNT_OF_RPC_ERROR", None)
+    os.environ.pop("GCS_CHECK_ALIVE_INTERVAL_SECONDS", None)
+    os.environ.pop("GCS_RETRY_CONNECT_INTERVAL_SECONDS", None)
```

Comment:
Can you make sure to restore the state properly here (set the value after the yield to the initial value).

Reply:
Sure, save the old value and restore it after yield is better. I will fix it.